semantics/language/translation/configuration.k

```diff
@@ -93,25 +86,31 @@ module C-CONFIGURATION
                </goto-loop-stack>
           </goto-calc>
      </goto-map-calc>
-     <control>
-          <curr-tu color="lightgray"> "" </curr-tu>
-          <curr-function color="lightgray">
-               file-scope
-          </curr-function>
-          <curr-program-loc color="black">
-               UnknownCabsLoc
-          </curr-program-loc>
-
-          // Toggle to allow writes and other special treatment for
-          // initializers.
-          <initializing> 0 </initializing>
-
-          <control-details> // TODO(chathhorn): dumb name
+     <elab-calc>
+          <elaborator multiplicity="*">
```

Comment:
Elaboration of function bodies could possibly be done in parallel (by saving the global state before each function), which I'd think would be desirable. But it could be done with like a stack too I suppose.